src/logs/index.js

```diff
@@ -0,0 +1,41 @@
+import Promise from 'bluebird'
+import AWS from '../util/aws'
+import { getLogGroup } from '../util/aws/cloudwatch-logs'
+import { getAliasVersion } from '../util/aws/lambda'
+import getLogs from '../util/get-logs'
+import { pkg } from '../util/load'
+import genName from '../util/generate-name'
+
+export default function (opts) {
+  const functionName = genName(opts.name).fullName
+  const aliasName = opts.stage
+  const stream = opts.stream
+  const region = opts.region || pkg().shep.region
+
+  AWS.config.update({ region })
+
+  // Should gracefully handle if function doesn't exist
+  return Promise.all([getLogGroup({ functionName }), getAliasVersion({ functionName, aliasName })])
+  .spread((logGroupName, functionVersion) => getLogs({ logGroupName, functionVersion, stream }))
+  .then(printEventsLoop)
+}
+
+function printEventsLoop ({ events, nextLogCall }) {
+  const rate = 250 // note this can't go under 200 as we're limited to 5 requests/sec
```

Comment:
Let's make this 500 out of the gate. Some AWS API limits are across the entire account, so would like to have a bigger buffer.